api/net/Areas/Admin/Controllers/ChartTemplateController.cs

```diff
@@ -147,7 +147,7 @@ public async Task<IActionResult> PreviewJsonAsync(ChartPreviewRequestModel model
             Template = model.Template,
             SectionSettings = model.Settings
         };
-        var chartTemplate = new TemplateEngine.Models.Reports.ChartEngineContentModel("test", chart, model.Content?.Select(c => new TemplateEngine.Models.Reports.ContentModel(c)));
+        var chartTemplate = new TemplateEngine.Models.Reports.ChartEngineContentModel("test", chart, model.Content?.Select(c => new TemplateEngine.Models.ContentModel(c)));
```

Comment:
Moved models to more generic location as they are used by more than one service.